src/webauth/utils.ts

```diff
@@ -633,19 +595,19 @@ export function verifyChallengeTxSigners(
 
   let serverSignatureFound = false;
   let clientSigningKeySignatureFound = false;
-  for (const signer of signersFound) {
+  signersFound.forEach((signer) => {
```

Comment:
Why is this preferred when it's technically slower due to add'l function call stacks?

Reply:
I'm not sure because you are correct about it being slower. The rule description is `   iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations  no-restricted-syntax`. In general I tried to change the for loops to something that made sense like filter but in some cases you do need to iterate over all which is why I just subbed in forEach. I could do an eslint ignore for those cases. For now I'll separate the webauth utils change into a separate PR.

Reply:
Oh wow interesting, good to know!